docs/resources/ntp_conf.md.erb

```diff
@@ -43,11 +43,13 @@ The following examples show how to use this Chef InSpec audit resource.
 
     describe ntp_conf do
       its('driftfile') { should eq '/var/lib/ntp/ntp.drift' }
-      its('server') { should eq [
+      its('server') do
+        should eq [
         0.ubuntu.pool.ntp.org,
```

Comment:
Shouldn't these be quoted?
Also, likely indented two more spaces?

Reply:
Thanks for catching this! Fixing now.